File: 240303/큰 숫자 자리수의 합/sum-of-large-numeric-digits.py
a = [int(x) for x in input().split()]

def f(a):
  if len(a) == 1:
    return a[0]
  return f(a[1:]) * a[0]

result = str(f(a))

num = 0
for i in range(len(result)):
  num += int(result[i])
print(num)

File: 240417/큰 숫자 자리수의 합/sum-of-large-numeric-digits.js
const fs = require('fs')
const numbers = fs.readFileSync(0).toString().trim().split(' ').map(Number)

let target = numbers[0] * numbers[1] * numbers[2]
let result = 0

function f(n) {
    if (n < 10) {
        result += n
        return result;
    }
    else {
        let a = n%10
        result += a
        return f(parseInt(n/10))
    }
}

console.log(f(target))